yoti-sdk-parent/pom.xml

```diff
@@ -102,7 +102,7 @@
     <!-- sdk dependency versions -->
     <slf4j.version>1.7.26</slf4j.version>
     <bouncy.castle.version>1.60</bouncy.castle.version>
-    <jackson.version>2.9.8</jackson.version>
+    <jackson.version>2.9.9.2</jackson.version>
```

Comment:
Is this deliberate?

Reply:
I think this came from a mistake in an interactive rebase (last commit in development before this branch was created), and this change should already be in the development branch so hopefully should be able to be merged without problem.

Reply:
Yep this is already in the development branch [see here](https://github.com/getyoti/yoti-java-sdk/commit/82f2ca7c1616291504221d8956a90801e63d44f8).  I can try to find a way to remove this from the PR, or we can just merge as is if it doesn't cause a problem